### ISSUE TITLE
Page & Comment size limits are restrictive

### ISSUE THREAD

**Issue by Zencat108** [free2z/free2z#8]
I'm trying to use Free2Z as a discussion forum, but that may be out of scope for this project.

I posted a page and offered to answer questions from the comments. I planned to update the main page with a Q&A section based on questions from the comments & my responses to make it easy to read. When I tried to update the page with my first Q&A, I ran into the 10,000 character limit.

So instead I tried to post my response to a comment, but that has an even shorter character limit. My response is 5200 characters; I didn't want to create a ton of comment entries because that gets hard to follow ... so perhaps I am too wordy for this site! :)

I love the concept of Free2Z, so willing to keep playing with it.
Any suggestions for a work-around are welcome.

I know the main focus is as a fundraising site, so for now I will direct people to my blog for answers to the questions they pose.
Also wanted to ask if there's a way to be notified if there are comments added to my page? (Not sure if that's feasible given the anon nature of the tech setup, which is cool.)

**Comment by skyl** [comment]
Hey, thanks for the feedback! I think we could easily go to 100,000. 10,000 was chosen arbitrarily without much thought. I actually assumed there would be more spammers trying to abuse the site so started pretty low. But, things are pretty orderly and mellow so far. I'll get the change in. I would suggest that it might be good to just add more pages and you could link to the subpages and maybe use the main page as an intro and table of contents. But, a single long page should also be an option. I want people to be able to do real, in-depth, long-form content. I have yet to hit the limit myself so figured it might be enough for everybody.

The current implementation of comments is onchain so they are constrained by the length of the Zcash private memo. But, I could see taking them offchain and then they could be longer. I was thinking about other micro-content that maybe instead of onchain they could be offchain and inexpensive- like 1,000 zaps for 0.01 (where a zap is like a Tweet).

Notifications could be possible - both in-app or over email or other means. I want people to have the option to create without giving personal information. But, we will probably add email accounts as an option for usages like this and eg password resets. Optional email addresses could also allow people to subscribe to pages or creators. We will also likely add ways to add social accounts so that creators could eg "login with Twitter/Github/etc".

I wouldn't say the main focus is necessarily fundraising. I think that vertical is very ripe for being disrupted. But, I definitely want people to feel comfortable posting arbitrary content with no explicit fundraising motive. So, thanks for stretching out and pushing the limits. "Free2z" means free to {any discrete place/point/goal}! So we definitely want to make your use case fit!

**Comment by skyl** [comment]
bumped to 100,000. Let me know if that works out for you!

**Comment by skyl** [comment]
added a note here to track optional email and notification of comments: https://github.com/free2z/free2z/projects/1#card-81035652
